modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,480 @@
+                            name: {{.Values.specialResourceModule.metadata.name}}
+                        subjects:
+                        - kind: ServiceAccount
+                          name: {{.Values.specialResourceModule.metadata.name}}
+                          namespace: {{.Values.specialResourceModule.spec.namespace}}
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: apps/v1
+                        kind: DaemonSet
+                        metadata:
+                            labels:
+                                app: {{ printf "%s-%s" .Values.specialResourceModule.metadata.name .Values.kernelFullVersion | replace "." "-" | replace "_" "-" | trunc 63 }}
+                            name: {{ printf "%s-%s" .Values.specialResourceModule.metadata.name .Values.kernelFullVersion | replace "." "-" | replace "_" "-" | trunc 63 }}
+                            namespace: {{.Values.specialResourceModule.spec.namespace}}
+                        spec:
+                            updateStrategy:
+                                type: OnDelete
+                            selector:
+                                matchLabels:
+                                    app: {{ printf "%s-%s" .Values.specialResourceModule.metadata.name .Values.kernelFullVersion | replace "." "-" | replace "_" "-" | trunc 63 }}
+                            template:
+                                metadata:
+                                    annotations:
+                                        scheduler.alpha.kubernetes.io/critical-pod: ""
+                                    labels:
+                                        app: {{ printf "%s-%s" .Values.specialResourceModule.metadata.name .Values.kernelFullVersion | replace "." "-" | replace "_" "-" | trunc 63 }}
+                                spec:
+                                    serviceAccount: {{.Values.specialResourceModule.metadata.name}}
+                                    serviceAccountName: {{.Values.specialResourceModule.metadata.name}}
+                                    containers:
+                                    - image: {{.Values.registry}}/{{.Values.specialResourceModule.metadata.name}}-{{.Values.groupName.driverContainer}}:{{.Values.kernelFullVersion}}
+                                      name: {{.Values.specialResourceModule.metadata.name}}
+                                      imagePullPolicy: Always
+                                      command: ["/sbin/init"]
+                                      lifecycle:
+                                        preStop:
+                                            exec:
+                                                command: ["/bin/sh", "-c", "systemctl stop kmods-via-containers@{{.Values.specialResourceModule.metadata.name}}"]
+                                            securityContext:
+                                                privileged: true
```

Comment:
This is also impacted by https://github.com/openshift/special-resource-operator/pull/229